src/components/Card.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Card.propTypes = {
+  image: PropTypes.node,
+  title: PropTypes.string,
+  ratingimage: PropTypes.node,
+  text: PropTypes.string,
+}
+
+export default function Card({ image, title, ratingimage, text }) {
```

Comment:
To keep it in camelCase: `ratingImage`
```suggestion
export default function Card({ image, title, ratingImage, text }) {
```